docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -0,0 +1,264 @@
+= Policy Configuration
+
+When `ec` command line requires configuration, e.g. for `ec validate image`
+command, the configuration can be provided as an inline JSON or YAML string,
+such as in this example:
+
+[,bash]
```

Comment:
I wonder if it would be good to mention the workaround we have for when the policy is on a local file, e.g. `ec validate image --policy "$(cat policy.yaml)". At least, until HACBS-1975 is place.

Reply:
I think we should add support for files and feel the pain in the meantime